api/Document.json

```diff
@@ -8169,17 +8177,11 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/onvisibilitychange",
           "support": {
-            "chrome": [
-              {
-                "version_added": "33"
-              },
-              {
-                "version_added": "13",
-                "prefix": "webkit"
-              }
-            ],
+            "chrome": {
+              "version_added": "62"
```

Comment:
OK, so one more thing to sort out with these `onfoo` event handler attributes. We also have `foo_event` entries, and `visibilitychange_event` still says 33, which was changed here. Can you update that entry too? It's fully possible that the event itself was supported earlier than the event handler attribute, and that there was a prefixed event.

There are also `afterscriptexecute_event` or `beforecriptexecute_event` entries in `Element`.

Might I suggest actually just handling these `onfoo` things separately, backing them out from this PR?

Reply:
It looks like the event itself was indeed supported earlier; the data  for `visibilitychange_event` is correct.  Although, it does need some mirroring -- I'll cherry-pick these changes and do it altogether in its own PR.

Reply:
Thanks!